.changes/next-release/bugfix-98fc2305-2aec-4bbd-974a-4bcaff6f1622.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Remote Lambda Run Config UI now displays correct functions in accordance with the AWS connection settings configured in the UI."
```

Comment:
This doesn't really describe what was fixed

Reply:
Would this be a better description? `Function selector in the Remote Lambda Run Config UI is now filled with requested lambda name even if it isn't in the active region.`

Reply:
Maybe: `Fixed the Remote Lambda Run Configuration periodically failing to load the list of functions`?

Reply:
Okay, got the difference between the two! Changed it in the next commit